src/Pos/Generator/Block/Mode.hs

```diff
@@ -317,7 +343,11 @@ instance MonadBlockGenBase m => MonadSlots (BlockGenMode m) where
     currentTimeSlotting = currentTimeSlottingSimple
 
 instance MonadBlockGenBase m => DB.MonadGState (BlockGenMode m) where
-    gsAdoptedBVData = gsAdoptedBVDataDefault
+    gsAdoptedBVData =
+        -- Currently we can't and don't want to test the system in boot mode
+        -- since in requires transaction to have specific distribution.
+        gsAdoptedBVDataDefault <&>
+        (\BlockVersionData{..} -> BlockVersionData { bvdUnlockStakeEpoch = 0, ..})
```

Comment:
Alright, i thought this PR should be merged ASAP but it can be WIP for one more day if you want to. I'll implement boot era payload corrections.